### PR TITLE
No handwritten bytes

### PR DESCRIPTION
**Pull request by matthiasbeyer**

### CODE REVIEW

File: mqtt-tester/src/client_report.rs [hunk 1]
```diff
@@ -95,26 +108,27 @@ async fn check_receiving_server_packet(client_exe_path: &Path) -> miette::Result
...
+            crate::command::ClientCommand::Send(
+                crate::util::paket_to_vec(MPacket::Subscribe({
+                    MSubscribe {
+                        id: MPacketIdentifier(1),
+                        subscriptions: MSubscriptionRequests {
+                            count: 1,
+                            data: &[b'a', b'/', b'b'],
+                        },
+                    }
+                }))
```
Comment (matthiasbeyer) [inline mqtt-tester/src/client_report.rs:129]: This might be wrong... please have a close look! :laughing:

File: mqtt-tester/src/client_report.rs [hunk 2]
```diff
@@ -95,26 +108,27 @@ async fn check_receiving_server_packet(client_exe_path: &Path) -> miette::Result
...
+            ),
+            crate::command::ClientCommand::Send(
+                crate::util::packet_to_vec(MPacket::Subscribe({
+                    MSubscribe {
+                        id: MPacketIdentifier(1),
+                        subscriptions: MSubscriptionRequests {
+                            count: 1,
+                            data: &[b'a', b'/', b'b'],
+                        },
+                    }
```
Comment (matthiasbeyer) [inline mqtt-tester/src/client_report.rs:128]: Still not sure about this bit.

Comment (TheNeikos) [reply mqtt-tester/src/client_report.rs:128]: Looks good to me. You should be able to write `b"a/b"` to get a byte string.

**Pull request merged**